functional/cypress-tests/cypress/fixtures/Widgets.js

```diff
@@ -10,7 +10,7 @@ module.exports = {
   aemHealthcheck: {
     name: 'AEM Healthcheck',
     endpoint: 'endpoint1',
-    schedulePeriod: '500',
+    schedulePeriod: '3',
```

Comment:
does it mean widget is refreshed each 3 millisecond during the test?

Reply:
every 3 seconds until it is deleted

Reply:
Are you sure it is 3 seconds? Does it mean it was set to 500 seconds previously?

Reply:
Yes I'm sure.